contributors/devel/sig-release/release.md

```diff
@@ -262,6 +247,8 @@ If you have an enhancement to target for an upcoming release milestone, begin a
 conversation with your SIG leadership and with that release's Enhancements
 Lead.
 
+Under special circumstanses you can fill an [exception](https://github.com/kubernetes/sig-release/blob/master/releases/EXCEPTIONS.md) request, in case you missed the release cycle deadline or an urgently needed enhancement cames up.
```

Comment:
circumstanses ==> circumstances
fill ==> file
"request, in case you missed" ==> "request—for example, if you miss ... enhancement comes up."